### ISSUE TITLE
Improve Accessibility When Creating/Updating API Keys

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1656]
The following could improve accessibility for creating and updating API keys:
- The checkbox label isn't read out by a screen reader when it's highlighted
- It should be made clear that the copy button copies the API key to the clipboard
- We could add an additional update button at the top of the list of permissions as well to save keyboard users have to tab through all permission. Alternatively, we could also make the existing button a landmark
- We could add a count of how many permissions there are so screen reader users would be less likely to get lost
